lib/Parse/ParseDecl.cpp

```diff
@@ -3763,6 +3792,18 @@ void Parser::ParseStructUnionBody(SourceLocation RecordLoc,
                       RecordLoc, TagDecl, FieldDecls,
                       T.getOpenLocation(), T.getCloseLocation(),
                       attrs.getList());
+  // Parse the deferred bounds expressions
+  for (const auto &Pair : deferredBoundsExpressions) {
+    EnterMemberBoundsExprRAII MemberBoundsContext(Actions);
+    DeclaratorDecl *FieldDecl = dyn_cast<DeclaratorDecl>(Pair.first);
```

Comment:
I looked at the code and I can just change all of this to be statically typed instead.

---
In reply to: [72877071](https://github.com/Microsoft/checkedc-clang/pull/23#discussion_r72877071) [](ancestors = 72877071)